RELEASE_NOTES.md

```diff
@@ -7,6 +7,14 @@ in the release. For a detailed list of changed components, refer to
 This document contains release notes for the current major release and all patches.
 For prior releases, see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 12.9.1 (2016-9-26)
+
+### Chef Server
+
+* Fix CHEFDK_GECODE_PATH for latest ChefDK
+* Fixed typo
+* Updated openssl
```

Comment:
In the 12.9.0 release notes in this file, we call out 12.9.1 as a release that is receiving a fix related to users acls.  Could you update that to 12.9.2 and add a note here that the users ACLs fix is delayed to 12.9.2 so that we can get the security fix out the door?